.gitignore

```diff
@@ -256,7 +256,16 @@ assets/jsconfig.json
 !.yarn/versions
 
 # local build files
+.backup_path
 local/bin/py/build/*
+local/bin/py/sh
+local/bin/py/js
+local/bin/js/
+local/bin/py/translations
+local/bin/py/add_notranslate_tag.py
+local/bin/py/placehold_translations.py
+local/bin/py/missing_metrics.py
+local/bin/py/submit_github_status_check.py
```

Comment:
should we also gitignore `local/bin/py/build/actions/integrations.py` ?

Reply:
also seeing a couple python files in my local tree that appear to not be gitignored under `local/bin/py/build`.  i.e. `content_manager.py`, `github_connect.py`, etc.